src/Domain/Behaviour/BehaviourGateway.php

```diff
@@ -254,4 +254,23 @@ public function selectMultipleStudentsOfOneIncident($gibbonMultiIncidentID)
 
         return $this->db()->select($sql, $data);
     }
+    public function queryFollowUpByBehaviourID($gibbonBehaviourID)
```

Comment:
Since this relates to the behaviour follow up, it'd be best located in the new BehaviourFollowupGateway you've created. For consistency, since it's not using a runQuery() method or query $criteria value, selectFollowUpByBehaviourID would be a better method name.